base/shell/explorer/traywnd.cpp

```diff
@@ -231,6 +231,7 @@ class CStartButton
 
         DWORD dwStyle = WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS | BS_PUSHBUTTON | BS_LEFT | BS_VCENTER;
 
+        // Hack
```

Comment:
It would seem we "loose" the reason why this is a hack. Is it still because the actual fix is to superclass? (as suggested at the top?) Or another reason?

Reply:
Superclassing is not relevant.